api/users/serializers.py

```diff
@@ -54,6 +55,7 @@ class UserSerializer(JSONAPISerializer):
     locale = HideIfDisabled(ser.CharField(required=False, help_text="User's locale, e.g.  'en_US'"))
     social = ListDictField(required=False)
     can_view_reviews = ShowIfCurrentUser(ser.SerializerMethodField(help_text='Whether the current user has the `view_submissions` permission to ANY reviews provider.'))
+    accepted_terms_of_service = ser.BooleanField(write_only=True, required=False)
```

Comment:
I think this being `write_only` is going to make it hard for the ember apps to show the banner for existing users.

Reply:
Discussed IRL: Ember OSF shouldn't be checking the API to determine whether or not to show the banner -- it's pushed as a status message from the OSF.